src/ORM/Connect/MySQLSchemaManager.php

```diff
@@ -330,7 +330,14 @@ public function indexList($table)
         $indexList = [];
 
         foreach ($indexes as $index) {
-            $groupedIndexes[$index['Key_name']]['fields'][$index['Seq_in_index']] = $index['Column_name'];
+            $column = $index['Column_name'];
+            if (isset($index['Collation']) && $index['Collation'] !== 'null') {
+                $column .= match ($index['Collation']) {
+                    'A' => ' ASC',
+                    'D' => ' DESC',
+                };
+            }
+            $groupedIndexes[$index['Key_name']]['fields'][$index['Seq_in_index']] = $column;
```

Comment:
This handles parsing MySQL's response when we ask it what indexes are on the table.
It has 3 options for `Collation`: "null" (no specified direction), "A" (Ascending), and "D" (Descending)
See https://dev.mysql.com/doc/refman/8.4/en/show-index.html#:~:text=How%20the%20column,not%20sorted).